pattoo/configuration.py

```diff
@@ -212,6 +214,122 @@ def ip_bind_port(self):
             result = int(intermediate)
         return result
 
+    def jwt_secret_key(self):
```

Comment:
Are these configuration parameters mentioned in the documentation?

Reply:
@Dev-Dominic